packages/web/src/scss/components/Alert/_Alert.scss

```diff
@@ -14,6 +14,10 @@
     border-radius: theme.$border-radius;
 }
 
+:where(.Alert) a {
+    color: inherit;
```

Comment:
And what about `active` and `hover` states? Shouldn't use proper emotion color as well?
But what would be appropriate solution? Emotion styles for links do not exist (which is probably correct), but maybe designers would have a different opinion and will be cool with current implementation… 

<img width="965" alt="image" src="https://github.com/lmc-eu/spirit-design-system/assets/33354913/2768b65f-3c43-4170-b7fe-fdde98a61b59">

Reply:
Oh my, you are right. We already have emotion colour states for hover and focus. So I added these. However the implementation of the generator is getting a bit wild, and I would like to discuss this in person tomorrow with @adamkudrna too.